pkgs/by-name/al/alacritty/package.nix

```diff
@@ -121,7 +122,10 @@ rustPlatform.buildRustPackage rec {
 
   dontPatchELF = true;
 
-  passthru.tests.test = nixosTests.terminal-emulators.alacritty;
+  passthru = {
+    tests.test = nixosTests.terminal-emulators.alacritty;
+    updateScript = nix-update-script { };
```

Comment:
Why is this necessary? Isn't it trivial to update this package with `nix-update` and `nixpkgs-update`?

Reply:
So that the bot can create the PRs automatically.

Reply:
Hmm, interesting. I thought an update script was required for that. I need to check in more detail what the bot does then.

Reply:
Yeah, I personally do use that. So if no one is really against, I'd rather leave it in.

Reply:
@r-vdp maybe add yourself as a maintainer than and drop my handle :)
I switched terminals.